data_tools/archive_project_datasets.sh

```diff
@@ -0,0 +1,21 @@
+set -e
+
+# Check that the correct number of arguments were provided.
+if [ $# -ne 3 ]; then
+    echo "Usage: ./archive_dataset.sh <crypto-token> <transfer-temp> <project-filter>"
```

Comment:
would `archive-folder` make more sense than `transfer-temp`?
After running this script, the dataset archives end up in this folder, hence I'm thinking that `transfer-temp` isn't a very good description.

Reply:
Right yes - because the upload script isn't automated. Could I ask you to take this PR off me and fix and land it? @marianamarasoiu

Reply:
Done, merging now!